components/Report/Report.module.css

```diff
@@ -0,0 +1,110 @@
+.container {
+    display: flex;
+    flex-direction: column;
+    margin: 1rem auto;
+    width: 80vw;
+    max-width: 1000px;
+}
+
+.container h1 {
+    text-align: center;
+    margin-block: 0rem;
+}
+
+.dashboardContainer {
```

Comment:
Not sure exactly which style, but we should have the same padding vertically and horizontally for titles in these cards; maybe consider lowering the vertical padding to match the amount of horizontal space we have? Note the upper vs left padding on "Race" and the border of that card, for example

<img width="87" alt="image" src="https://github.com/Kevinjchang98/St-Francis/assets/17811745/c2bf22f0-d776-4824-9c0d-735d492cfb75">

Reply:
fixed!